docs/statistics.md

```diff
@@ -111,11 +119,31 @@ The time elapsed, in milliseconds, since the SRT socket has been created (after
 
-The total number of sent data packets, including retransmitted packets. Available for sender.
+The total number of sent DATA packets, including retransmitted packets ([pktRetransTotal](#pktRetransTotal)). Available for sender.
+
+In case the `SRTO_PACKETFILTER` socket option is enabled (refer to [API.md](API.md)), this statistic counts packet filter control packets ([pktSndFilterExtraTotal](#pktSndFilterExtraTotal)) as well. Introduced in SRT v1.4.0.
 
 ### pktRecvTotal
 
-The total number of received packets, including retransmitted packets. Available for receiver.
+The total number of received DATA packets, including retransmitted packets ([pktRcvRetransTotal](#pktRcvRetransTotal)). Available for receiver.
+
+In case the `SRTO_PACKETFILTER` socket option is enabled (refer to [API.md](API.md)), this statistic counts packet filter control packets ([pktRcvFilterExtraTotal](#pktRcvFilterExtraTotal)) as well. Introduced in SRT v1.4.0.
+
+### pktSentUniqueTotal 
+
+The number of unique DATA packets sent by the SRT sender. Available for sender. 
+
+This value contains only unique **original** DATA packets. Retransmitted DATA packets ([pktRetransTotal](#pktRetransTotal)) are not taken into account. In case the `SRTO_PACKETFILTER` socket option is enabled (refer to [API.md](https://cac-word-edit.officeapps.live.com/we/API.md)), packet filter control packets ([pktSndFilterExtraTotal](https://cac-word-edit.officeapps.live.com/we/wordeditorframe.aspx?ui=en-GB&rs=en-US&wopisrc=https%3A%2F%2Fhai365.sharepoint.com%2Fsites%2FTeam-SRT%2F_vti_bin%2Fwopi.ashx%2Ffiles%2F421d054732ce45e08814ced46f033282&wdenableroaming=1&mscc=1&hid=53334D9F-F03D-B000-138D-338011C1234D&wdorigin=Other&jsapi=1&newsession=1&corrid=8ef94636-16cd-4a4f-ae0b-da2e3f2cacab&usid=8ef94636-16cd-4a4f-ae0b-da2e3f2cacab&instantedit=1&wopicomplete=1&wdredirectionreason=Unified_SingleFlush#pktSndFilterExtraTotal)) are not taken into account as well. 
+
+By implementation, this value corresponds to the number of original DATA packets sent by the SRT sender and can be calculated as follows: `pktSentUniqueTotal = pktSentTotal – pktRetransTotal` or `pktSentUniqueTotal = pktSentTotal – pktRetransTotal - pktSndFilterExtraTotal` if the  `SRTO_PACKETFILTER` socket option is enabled. The original packets are sent only once. 
+
+### pktRecvUniqueTotal 
+
+The number of unique original or retransmitted DATA packets received by the SRT receiver in time. Available for receiver. 
+
+This value contains only **unique** meaning first arrived DATA packets so that there is no difference whether it is original or retransmitted (in case of loss) packet that comes first and is taken into account. This statistic doesn't count the packets received as duplicated (retransmitted or sent several times by the defective hardware/software) or “too late to deliver” (retransmitted or reordered and as a result arrived too late). See [pktRcvDropTotal](#pktRcvDropTotal) statistic which counts “too late to deliver” packets.
```

Comment:
```suggestion
This value contains only **unique** (meaning "first arrived") DATA packets. There is no difference whether a packet is original or retransmitted (in case of loss). Whichever packet comes first is taken into account. This statistic doesn't count *duplicate* packets (retransmitted or sent several times by defective hardware/software) or *retransmitted packets* that are reordered and, as a result, arrive too late to deliver. See the [pktRcvDropTotal](#pktRcvDropTotal) statistic, which counts “too late to deliver” packets.
```

Reply:
@stevomatthews Changed to

This value contains only **unique** (meaning "first arrived") DATA packets. There is no difference whether a packet is original or retransmitted (in case of loss). Whichever packet comes first is taken into account. This statistic doesn't count duplicate packets (retransmitted or sent several times by defective hardware/software) or arrived too late packets (retransmitted or original packets arrived out of order) and, as a result, dropped by the TLPKTDROP mechanism (see [pktRcvDropTotal](#pktRcvDropTotal) statistic).

Reply:
Good.